README.md

```diff
@@ -45,7 +45,20 @@ The `ntripping` utility has the following usage:
             --url <url>             [default: na.skylark.swiftnav.com:2101/CRS]
 
 Different resources can be requested from different locations. By default, a San
-Francisco latitude, longitude, and height will be used.
+Francisco latitude, longitude, and height will be used. The data returns is in RTCM format.
+
+The `--url` must be formatted as such: `{username}:{passsword}@{area}.skylark.swiftnav.com:{port}/{mountpoint}`, 
+where `username` and `password` refer to the user's [device registry](https://device-registry.cs.swiftnav.com/#/)
+username and password. The full list of available endpoinds and mountpoints can be found 
+[here](https://swift-nav.atlassian.net/wiki/spaces/ENG/pages/399902820/Skylark+Endpoints+Mountpoints).
```

Comment:
Let's move the device registry and skylark specific mointpoint instructions to an internal wiki and point to it from here (noting that it's Swift internal just in case a customer sees and tries the link).